shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/helpers/extensions/ios/UITestLogout.kt

```diff
@@ -1,15 +1,13 @@
 package co.nimblehq.blisskmmic.helpers.extensions.ios
 
-import co.nimblehq.blisskmmic.data.database.datasource.LocalDataSourceImpl
-import com.russhwolf.settings.Settings
+import co.nimblehq.blisskmmic.data.database.datasource.LocalDataSource
 import org.koin.core.component.KoinComponent
 import org.koin.core.component.inject
 
 object UITestLogout: KoinComponent {
```

Comment:
Just curious, why this file `UITestLogout.kt` is a kotlin file but it is under the `ios` extension? Shouldn't it be in `android` extension instead? 🤔

Reply:
This is a helper object for iOS to use, hence there's iOS in the package. It is under `common`.